src/utils.py

```diff
@@ -209,6 +230,58 @@ def flatten(nested_iterable: Iterable[Iterable[T]]) -> chain[T]:
     return chain.from_iterable(nested_iterable)
 
 
+def subprocess_kwargs():
+    """
+    Create a set of arguments which make a ``subprocess.Popen`` (and
+    variants) call work with or without Pyinstaller, ``--noconsole`` or
+    not, on Windows and Linux.
+    Typical use:
+      subprocess.call(['program_to_run', 'arg_1'], **subprocess_args())
+    ---
+    Originally found in https://github.com/madmaze/pytesseract/blob/master/pytesseract/pytesseract.py
+    Recipe from https://github.com/pyinstaller/pyinstaller/wiki/Recipe-subprocess
+    which itself is taken from https://github.com/bjones1/enki/blob/master/enki/lib/get_console_output.py
+    """  # noqa: D415,D400
```

Comment:
The fix for this `D4XX` is to simply add a newline after the first.
You can also use code blocks in docstrings
![image](https://github.com/Toufool/AutoSplit/assets/1350584/56b40647-f8da-4f0d-9563-e0f53f9bd70d)


```suggestion

    Typical use:
    ```python
    subprocess.call(['program_to_run', 'arg_1'], **subprocess_args())
    ```
    ---
    Originally found in https://github.com/madmaze/pytesseract/blob/master/pytesseract/pytesseract.py
    Recipe from https://github.com/pyinstaller/pyinstaller/wiki/Recipe-subprocess
    which itself is taken from https://github.com/bjones1/enki/blob/master/enki/lib/get_console_output.py
    """
```